prow/external-plugins/cherrypicker/server.go

```diff
@@ -289,12 +289,13 @@ func (s *Server) handlePullRequest(l *logrus.Entry, pre github.PullRequestEvent)
 		if len(cherryPickMatches) == 0 || len(cherryPickMatches[0]) != 2 {
```

Comment:
Don't we need to move the second check into the loop?

Reply:
I think we can remove it completely... either it matches and match[1] exists, or it doesn't and we won't iterate.

Reply:
I mean this would be silly, right?
```
			if len(match) != 2 {
				continue
			}
```

Reply:
I guess you are right, if it matches the capture group will have a value so we can just remove the check